### PR TITLE
added in README file the database instructions

### PR DESCRIPTION
**Pull request by Motopshyco**
README file actualized

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,4 +149,88 @@ As reviewers, a PR should be only approved if following conditions are met (In o
...
+  // Alternatively, you can use spread operator in ES6 syntax
+  // const newRegions = ['south_carolina', 'texas']
+  // regions: admin.firestore.FieldValue.arrayUnion(...newRegions)
+});
+
+5. delete a query:
+const res = await db.collection('cities').doc('ID').delete();
+
+6. get a query:
+const cityRef = db.collection('cities').doc('ID');
```
Comment (alexeindev) [inline README.md:230]: I don't think that it's a good idea to put the logic and the way we do the queries on the README file for security reasons

File: README.md [hunk 2]
```diff
@@ -149,4 +149,88 @@ As reviewers, a PR should be only approved if following conditions are met (In o
 - git add \<file or folder\>
 - git commit -m \<meaningful commit message\>
 - git push
-- git pull
\ No newline at end of file
+- git pull
+
+## Database connection
+1. Run the npm install
```
Comment (alexeindev) [inline README.md:155]: The npm install is already specified but the other step looks good 👍🏻

**Review by alexeindev — CHANGES_REQUESTED** [review]

**Review by daniel33gomez — CHANGES_REQUESTED** [review]
May the examples be about the actual collection and json scencario object?

File: README.md (hunk 1 above)
Comment (Motopshyco) [reply README.md:230]: that is an example about the database works and everyone can search the same structure.

**Review by daniel33gomez — APPROVED** [review]

**Comment by Sam-sta** [comment]
> May the examples be about the actual collection and json scencario object?

Daniel that's not a good idea

File: README.md [hunk 3]
```diff
@@ -149,4 +149,88 @@ As reviewers, a PR should be only approved if following conditions are met (In o
...
+2. configure the service account (in first iterations we will use a default credetial)
```
Comment (Sam-sta) [inline README.md:156]: Mateo here we can maybe explain better how to use the credential, tomorrow we can do it

**Review by Sam-sta — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (Motopshyco) [reply README.md:156]: yeah but for now we don't need it, then we will add it tomorrow

File: README.md (hunk 1 above)
Comment (J0c4) [reply README.md:230]: Commonly, README file should contain information about using, setting up, configuring or  understanding the project. Its target audencie is intended to be potential contributors, users or any other people trying to learn about what it does. The `query examples` section here could be more appropriate for a project wiki or database library documentation.

This actually should include just instructions about how the project should be configured for running. Example
```
Setup configuration file:

/config.json

{
    "serviceAccount": <account example>,
    "apiKey": <apiKey>
}

To be provided with an account/key, ask a project admin to provide this information
```

**Review by Sam-sta — APPROVED** [review]

**Review by daniel33gomez — CHANGES_REQUESTED** [review]
Please include the local path we should save the key. In order to have consistency

**Comment by daniel33gomez** [comment]
> Please include the local path we should save the key. In order to have consistency

Or maybe instruction related to the authentication

File: README.md [hunk 4]
```diff
@@ -174,13 +174,29 @@ As reviewers, a PR should be only approved if following conditions are met (In o
...
+- git pull
+
+## Database connection
+1. Configure the service account (in first iterations we will use a default credential)
+2. Ask for the database key and install it.
+3. Create a folder inside the folder where you put the project cloned folder and named it keys.
+4. Inside the keys folder move the json key.
+5. For initilize the app:
+
+import serviceAccount from `'../../../keys/metal-slug-maker-firebase-adminsdk-8dsbv-e517ebfe1a.json'`;
```
Comment (daniel33gomez) [inline README.md:194]: I think it should be in the databaseLibrary file

**Review by daniel33gomez — CHANGES_REQUESTED** [review]
Please check my comment and remove the package-lock.json from this merge

File: README.md (hunk 4 above)
Comment (Sam-sta) [reply README.md:194]: No, but i will skip the key file name

**Review by daniel33gomez — APPROVED** [review]

**Pull request merged**